### PR TITLE
Multiline filters 02 22 21

### PR DESCRIPTION
**Pull request by mikeglauser**
## Proposed changes

Created a topic about creating multiline filters, per the acceptance criteria in https://jira.splunk.com/browse/ADDON-22623

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](https://github.com/splunk/splunk-connect-for-kubernetes/blob/develop/CONTRIBUTING.md) doc
- [ ] I have read the [CLA](https://github.com/splunk/splunk-connect-for-kubernetes/blob/develop/CLA.md)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,9 +152,33 @@ Some parameters used with Splunk Connect for Kubernetes can have an impact on ov
 
 Splunk Connect for Kubernetes can exceed the default throughput of HEC. To best address capacity needs, Splunk recommends that you monitor the HEC throughput and back pressure on Splunk Connect for Kubernetes deployments and be prepared to add additional nodes as needed.
 
-# Processing Multi-Line Logs
+# Processing multiline Logs
 
-One possible filter option is to enable the processing of multi-line events. This feature is currently experimental and considered to be community supported.
+One possible filter option is to enable the processing of multiline events. This feature is currently experimental and considered to be community supported.
+
+# Configuring multiline fluentd filters to line break multiline logs
```
Comment (vinzent) [inline README.md:159]: isn't this already handled with the `customFilters` param ? https://github.com/splunk/splunk-connect-for-kubernetes/blob/develop/helm-chart/splunk-connect-for-kubernetes/values.yaml#L428

File: README.md [hunk 2]
```diff
@@ -152,9 +152,33 @@ Some parameters used with Splunk Connect for Kubernetes can have an impact on ov
...
+        timeout_label @SPLUNK
+        stream_identity_key stream
+        multiline_start_regexp /^\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}|^\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}|^\d{1,3}.\d{1,3}.\d{1,3}.\d{1,3}\s-\s-/
+        multiline_end_regexp /\\n$/
+        separator ""
+        flush_interval 5s
+</filter>
+```
+
+2. Add the multiline filter to your deployment's [logging configmap.](https://github.com/splunk/splunk-connect-for-kubernetes/blob/develop/manifests/splunk-kubernetes-logging/configMap.yaml)
```
Comment (vinzent) [inline README.md:179]: this ConfigMap is managed by helm and will be overriden on next helm update.

**Review by vinzent — COMMENTED** [review]
can't access https://jira.splunk.com/browse/ADDON-22623 (or https://jira.splunk.com/ at all). is it Splunk internal?

**Comment by vinzent** [comment]
my suggestion: https://github.com/splunk/splunk-connect-for-kubernetes/pull/570

File: README.md (hunk 1 above)
Comment (chaitanyaphalak) [reply README.md:159]: the idea is to use the example filter with customFilters param

File: README.md (hunk 2 above)
Comment (chaitanyaphalak) [reply README.md:179]: we can change this step to mention using customFilters param

**Comment by mikeglauser** [comment]
Hi @chaitanyaphalak ,
I have made an edit to this commit in order to capture mention of the customFilters parameter. Please let me know what you think, as well as any possible next courses of action.

**Review by rockb1017 — APPROVED** [review]

**Pull request merged**